packages/cashier/src/stores/account-transfer-store.ts

```diff
@@ -521,12 +521,20 @@ export default class AccountTransferStore {
         } else if (
             (selected_from?.is_mt && this.selected_to.is_mt) ||
             (selected_from?.is_dxtrade && this.selected_to.is_dxtrade) ||
+            (selected_from?.is_ctrader && this.selected_to.is_ctrader) ||
             (selected_from?.is_dxtrade && this.selected_to.is_mt) ||
-            (selected_from?.is_mt && this.selected_to.is_dxtrade)
+            (selected_from?.is_dxtrade && this.selected_to.is_ctrader) ||
+            (selected_from?.is_ctrader && this.selected_to.is_dxtrade) ||
+            (selected_from?.is_ctrader && this.selected_to.is_mt) ||
+            (selected_from?.is_mt && this.selected_to.is_dxtrade) ||
+            (selected_from?.is_mt && this.selected_to.is_ctrader)
         ) {
             // not allowed to transfer from MT to MT
             // not allowed to transfer from Dxtrade to Dxtrade
-            // not allowed to transfer between MT and Dxtrade
+            // not allowed to transfer from cTrader to cTrader
+            // not allowed to transfer between MT and Dxtrade and visa versa
+            // not allowed to transfer between Dxtrade and cTrader and visa versa
+            // not allowed to transfer between MT and cTrader and visa versa
             // if new value of selected_from is different from selected_to
```

Comment:
I appreciate aesthetics of such a large conditional expression :D :D :D

Reply:
yeah @wojciech-deriv me too
actually there are many things need refactoring in legacy account-transfer-store, but since we are going to replace it with version 2 i decided to leave it as it is 🤔